src/app/components/ExternalLinksCard.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  Title,
+  Bullseye,
+  Split,
+  Flex,
+  FlexItem,
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  Button,
+  Grid,
+  GridItem,
+  CardTitle
+} from '@patternfly/react-core'
+
+
+import redhat_logo from "@app/bgimages/redhat_clear.png";
+import aws_logo from "@app/bgimages/aws_clear.png";
+import azure_logo from "@app/bgimages/azure_clear.png";
+import google_logo from "@app/bgimages/google_clear.png";
+
+
+const providerInfos = [
+  {
+    name: 'aws',
+    text: 'Launch in AWS ->',
+    url: 'https://aws.amazon.com/marketplace/pp/prodview-kv5mi3ksb2mma?sr=0-1&ref_=beagle&applicationId=AWSMPContessa'
+  },
+  {
+    name: 'google',
+    text: 'Launch in Google Cloud ->',
+    url: 'https://console.cloud.google.com/marketplace/product/rhel-cloud/rhel-9?project=cockpituous'
+  },
+  {
+    name: 'azure',
+    text: 'Launch in Azure ->',
+    url: 'https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-20190605?tab=Overview'
+  }
+]
+
+const loadImage = (name) => {
+  switch (name) {
+    case 'aws':
+      return aws_logo
+    case 'azure':
+      return azure_logo
+    case 'google':
+      return google_logo
+  }
+}
+
+const ExternalLinksCard: React.FunctionComponent = () => {
+  return (
+    <Flex direction={{default: 'column'}} flex={{default: 'flex_3'}}
+          justifyContent={{default: 'justifyContentSpaceBetween'}}
+          style={{border: '1px solid #D2D2D2', padding: '2%'}}>
```

Comment:
Ugh I find inline styles pretty ugly. That's a preference obviously - but I think code is much cleaner, if you use CSS files for adding the styles, and use classNames for the selectors.

Reply:
I'm not super familiar with Typescript best practices but @epapbak if you want to go this route I would be okay with it.
@KKoukiou what is considered good style, adding them into a single app.css or should this be split into component css files?

Reply:
I prefer one CSS file per page, but if you have very little custom CSS (which should be your goal) we can keep the whole CSS in a single file.

Reply:
I've tried to use different layouts offered by Patternfly, but the end result was always slightly different from what I expected.

For example, aligning the "Build an image ->" link with the other links was pretty much impossible without using `Flex` layout with `justifyContent={{default: 'justifyContentSpaceBetween'}}` and `alignSelf={{default: "alignSelfStretch"}}` on each child. But I'm a beginner with this framework, so there might be something I missed while trying it out...

Reply:
Regarding the layout, I wonder if you might have more luck using a grid?

https://www.patternfly.org/v4/layouts/grid#with-gutters

You should be able to specify a span of 4 for the first card and 8 for the second card.

Reply:
I did spend an entire afternoon trying to do it with the Grid and Split layouts. Didn't try Level though. I'll give it a go when I have some time, see if I can make it work without additional styling.

Reply:
Oof, yes alignment can be tricky. There's a reason centering a div is a meme... :sob: :wink: 

If I can find some time this afternoon I'll try a few things out, too.

Reply:
This seems to work reasonably well:
```typescript
        <Grid hasGutter>
          <GridItem md={4} sm={12}>
            <InternalLinksCard />
          </GridItem>
          <GridItem md={8} sm={12}>
            <ExternalLinksCard />
          </GridItem>
        </Grid>

```